### DISCUSSION TITLE
请问sign参数怎么关闭

### DISCUSSION THREAD

**Discussion by mobeicanyue** [AlistGo/alist#5901] [General]
我之前装alist的时候，点击文件下载链接，没有带查询参数
现在点击会带 https://xxxxx.xxx/d/show.jpeg?sign=hdjevvdjidjdvejksW0HXyfD67A=:0
我不知道配置了什么，都开启了游客访问，请问可以关闭这个参数吗。

**Comment by Dragon1573** [comment]
@mobeicanyue - 我看到你在7个月前把这个 Discussion 给 Close 掉了，是已经自己解决了嘛？是否可以提供一下你的解决方案？

**Reply by Dragon1573** [reply]
已解决❤️

1. 登录站点后台，导航到 [设置 > 全局](https://alist.example.com/@manage/settings/global) 下，关闭「签名所有」。
2. 导航到 [存储](https://alist.example.com/@manage/storages) ，编辑资源所在的存储路径，关闭「启用签名」。
3. 导航到 [元信息](https://alist.example.com/@manage/metas) ，确保资源所在的存储路径没有设置元信息。

- 参见 https://github.com/alist-org/alist/discussions/6024#discussioncomment-8448145
